samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -71,12 +71,27 @@
     /* "Key": // dotnet user-secrets set "AzureSpeech:Key" "MY_AZURE_SPEECH_KEY" */
   },
 
+  "Auth": {
+    //
+    // Auth to use to gate access to exposed API's.
+    // - Supported Types are "None", "ApiKey", or "AzureAd".
+    //
+    "Type": "None",
+    "ApiKey": "", // The API key to use for auth - use user secrets, environment variable or Key Vault to populate
```

Comment:
I was trying to avoid comments after properties in favor of using the comment block above (as much as possible, anyway).